stackit/stackit_core.py

```diff
@@ -109,6 +109,17 @@ def _search(config):
             print_question(question, count)
             if count % NUM_RESULTS == 0:
                 focus_question(question_logs)
+    if len(questions) == 0:
+            click.echo(
+                click.style("Your search \'{0}\' with tags \'{1}\' returned no results.".format(config.term,config.tag),
+                fg="red"))
+            user_input = click.prompt("Enter b to launch browser or q to quit")
+            if user_input == "q":
+                sys.exit()
+            elif user_input == "b":
+                click.launch(
+                    "https://stackoverflow.com/search?q={0}{1}".format(config.term,config.tag))
+    
```

Comment:
@grobbie94 

- PEP8 [Separate top-level function and class definitions with two blank lines.](https://www.python.org/dev/peps/pep-0008/#blank-lines)
- if a user hit the `x` key nor than `q` or `b` stackit will exit with no explication... I think it could be better to do something like:
```python
if not questions:
    while True: 
        click.echo(
            click.style("Your search \'{0}\' with tags \'{1}\' returned no results.".format(config.term,config.tag),
            fg="red"))
        user_input = click.prompt("Enter b to launch browser or q to quit")
        if user_input == "q":
            sys.exit() 
        elif user_input == "b":
            click.launch(
                "https://stackoverflow.com/search?q={0}{1}".format(config.term,config.tag))
        else:
            click.echo(click.style(
                "The input entered was not recognized as a valid choice.",
                fg="red"))
```